.github/workflows/docker-build-push.yaml

```diff
@@ -0,0 +1,91 @@
+name: Docker build & push image
```

Comment:
This will be needed in a forthcoming PR for doing E2E testing in ephemeral CI instances. 

The reason being it will be much easier to track regressions and also we need the container env anyway for network tests.